examples/wav2vec/README.md

```diff
@@ -199,6 +199,48 @@ loss = model(input_values, labels=labels).loss
 loss.backward()
 ```
 
+## Deploying wav2vec 2.0 with torchaudio
+
+`torchaudio` has added wav2vec 2.0 model definition that supports TorchScript, alongside of function to import model instances from `fairseq` or 🤗Transformers. By using TorchScript, you can deploy your wav2vec 2.0 model to ONNX runtime, [C++](https://github.com/pytorch/audio/tree/master/examples/libtorchaudio/speech_recognition), [iOS](https://github.com/pytorch/ios-demo-app/tree/master/SpeechRecognition), and [Android](https://github.com/pytorch/android-demo-app/tree/master/SpeechRecognition).
```

Comment:
Examples for iOS and Android are being updated to use torchaudio.

- https://github.com/pytorch/android-demo-app/pull/141
- https://github.com/pytorch/ios-demo-app/pull/53

ONNX support via TorchScript is reported to work here
- https://github.com/pytorch/fairseq/issues/3010